data/json/items/armor/torso_clothes.json

```diff
@@ -726,7 +726,7 @@
     "type": "ARMOR",
     "name": { "str": "leather vest" },
     "description": "A vest made from thick leather.  Offers excellent protection from cuts.",
-    "weight": "997 g",
+    "weight": "1530670 mg",
```

Comment:
You have way too many things here where you're changing from a reasonable gram value (or in this case one that should have just bean 1kg) and changing it to ridiculous milligram values

Reply:
This value is the result of the cleaning of those two recipes:
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/recipes/armor/torso.json#L1159
https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/recipes/armor/pets_dog.json#L41

I understand that the precise miligram value is less readable than gram.
Would you like this value to be changed to 1530 g?

Reply:
We don't need milligram precision, but we should either adjust the recipe or just bump this is 1530g for weight accuracy.

Reply:
Yes, I would say anything that is more than 5 grams should be rounded to the nearest gram, if you want to do it automatically.